src/gc.c

```diff
@@ -2855,7 +2944,7 @@ static int _jl_gc_collect(jl_ptls_t ptls, jl_gc_collection_t collection)
         gc_invoke_callbacks(jl_gc_cb_root_scanner_t,
             gc_cblist_root_scanner, (collection));
     }
-    gc_mark_loop(ptls);
+    gc_mark_loop_master(ptls);
```

Comment:
Should we use `spinmaster` throughout as aterm?

Reply:
I don't think so: we're not using that protocol in this PR.